src/searchbar/SearchBar-android.js

```diff
@@ -27,7 +27,7 @@ class SearchBar extends Component {
   clear = () => {
     this.input.clear();
     this.onChangeText('');
-    this.props.onClearText();
+    this.props.onClear && this.props.onClear();
```

Comment:
Can just use `this.props.onClear()` since we already set defaultProps to have a function

Reply:
Meh, didn't commit that lol